_posts/2016-07-14-official-docker-image.md

```diff
@@ -0,0 +1,181 @@
+---
+title: Official Docker image
+author: Flavio Castelli
+layout: blogpost
+---
+
+I'm pleased to announce the first release of a production ready Docker image for
+Portus. This has been one of the most requested features since the creation of
+this project.
+
+We always had a Docker image available, but that was oriented toward developers,
+hence using it in production would have been sub-optimal.
+
+The next paragraphs illustrate how the image has been created and the topics we
+focused on.
+
+## Security
+
+SUSE's containers team cares about security, hence we made the following
+decisions:
+
+  * This image enforces the usage of HTTPS to access Portus.
```

Comment:
What about kubernetes users, who terminate TLS at ingress? Or plain AWS users, who terminate TLS at the ELB?